docs/docs-beta/docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,177 @@
+</details>
+
+## Step 1: Set up the Dagster environment
+
+1. Open the terminal and create a new directory for your project:
+
+   ```bash
+   mkdir dagster-quickstart
+   cd dagster-quickstart
+   ```
+
+2. Create and activate a virtual environment:
+
+   <Tabs>
+   <TabItem value="macos" label="MacOS">
+   ```bash
+   python -m venv venv
+   source venv/bin/activate
+   ```
+   </TabItem>
+   <TabItem value="windows" label="Windows">
+   ```bash
+   python -m venv venv
+   source venv\Scripts\activate
+   ```
+   </TabItem>
+   </Tabs>
+
+3. Install Dagster and the required dependencies:
+
+   ```bash
+   pip install dagster dagster-webserver pandas
+   ```
+
+## Step 2: Create the Dagster project structure
+
+:::info
+The project structure in this guide is simplified to allow you to get started quickly. As a next step, check out the [ETL pipeline tutorial](/tutorial/tutorial-etl) to learn how to build more complex pipelines using best practices.
+:::
+
+Next, you'll create a basic Dagster project that looks like this:
+
+```
+dagster-quickstart/
+├── quickstart/
+│   ├── __init__.py
+│   └── assets.py
+├── data/
+    └── sample_data.csv
+```
+
+1. To create the files and directories outlined above, run the following:
+
+   ```bash
+   mkdir quickstart data
+   touch quickstart/__init__.py quickstart/assets.py
+   touch data/sample_data.csv
+   ```
+
+2. In the `data/sample_data.csv` file, add the following content:
+
+   ```csv
+   id,name,age,city
+   1,Alice,28,New York
+   2,Bob,35,San Francisco
+   3,Charlie,42,Chicago
+   4,Diana,31,Los Angeles
+   ```
+
+   This CSV will act as the data source for your Dagster pipeline.
+
+## Step 3: Define the assets
+
+Now, create the assets for the ETL pipeline. Open `quickstart/assets.py` and add the following code:
+
+```python
+import pandas as pd
+import dagster as dg
+
+
+@dg.asset
+def processed_data():
+    ## Read data from the CSV
+    df = pd.read_csv("data/sample_data.csv")
+
+    ## Add an age_group column based on the value of age
+    df["age_group"] = pd.cut(
+        df["age"], bins=[0, 30, 40, 100], labels=["Young", "Middle", "Senior"]
+    )
+
+    ## Save processed data
+    df.to_csv("data/processed_data.csv", index=False)
+    return "Data loaded successfully"
+
+
+## Tell Dagster about the assets that make up the pipeline by
+## passing it to the Definitions object
+## This allows Dagster to manage the assets' execution and dependencies
+defs = dg.Definitions(assets=[processed_data])
+```
+
+This may seem unusual if you're used to task-based orchestration. In that case, you'd have three separate steps for extracting, transforming, and loading.
+
+However, in Dagster, you'll model your pipelines using assets as the fundamental building block, rather than tasks.
+
+## Step 4: Run the pipeline
+
+1. In the terminal, navigate to your project's root directory and run:
+
+   ```bash
+   dagster dev -f quickstart/assets.py
```

Comment:
hm, technically I don't think we need `quickstart/__init__.py` if we are running the file directly.

If we are creating a module, we may instead opt to just run `dagster dev`

Reply:
That's true. D'you think there's value in having a file-specific command here? I wonder if users seeing only `dagster dev` may think it will work anywhere, when in reality there are some requirements to get it to work. I may be overthinking that though.

Reply:
I like using `-f` but maybe instead of making a module, we just have the user create a `quickstart.py` file, and `dagster dev -f quickstart.py`? Unless you see benefit in creating the basic project structure for it to grow?

Reply:
Yeah, using a simplified structure here is probably a good thing as it'll get users used to working in similar project structures. We're not explaining it, so I think it's fine to keep.